src/shared/helpers/getActualDataList.ts

```diff
@@ -1,4 +1,4 @@
-import { dayJS } from '@/app/services/dayjs';
+import { dayJS } from '../dayjs';
```

Comment:
```suggestion
import { dayJS } from '@/shared/helpers/dayJS';
```

Reply:
but as @natanchik mentioned, we need this service only for tests, and should use lib in code by direct import 
```
import dayjs from 'dayjs';
```

Reply:
At the last meeting, we decided to use only one specific import to avoid confusion